src/views/ipv/pyi-triage-select-device.njk

```diff
@@ -0,0 +1,43 @@
+{% extends "shared/base.njk" %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/radios/macro.njk" import govukRadios %}
+{% set pageTitleName = 'pages.pyiTriageSelectDevice.title' %}
+{% set googleTagManagerPageId = "pyiTriageSelectDevice" %}
+
+{% set errorState = pageErrorState %}
+{% set errorTitle = 'pages.pyiTriageSelectDevice.content.formErrorMessage.errorSummaryTitleText' | translate %}
+{% set errorText = 'pages.pyiTriageSelectDevice.content.formErrorMessage.errorSummaryDescriptionText' | translate %}
+{% set errorHref = "#pyiTriageSelectDeviceForm" %}
+
+{% block content %}
+  <h1 class="govuk-heading-l" id="header" data-page="{{ googleTagManagerPageId }}">{{ 'pages.pyiTriageSelectDevice.header' | translate }}</h1>
+  <p class="govuk-body">{{ 'pages.pyiTriageSelectDevice.content.paragraph1' | translate }}</p>
+
+  <form id="pyiTriageSelectDeviceForm" action="/ipv/page/{{ pageId }}" method="POST">
+    <input type="hidden" name="_csrf" value="{{ csrfToken }}">
+    {% set radiosConfig = {
+      idPrefix: "journey",
+      name: "journey",
+      items: [
+        {
+          value: "computer-or-tablet",
+          text: 'pages.pyiTriageSelectDevice.content.formRadioButtons.onComputerOrTabletButtonText' | translate
+        },
+        {
+          value: "smartphone",
+          text: 'pages.pyiTriageSelectDevice.content.formRadioButtons.onSmartphoneButtonText' | translate
+        }
+      ]
+    } %}
+
+    {% if errorState %}
+      {% set errorMessageObject = { 'text': 'pages.pyiTriageSelectDevice.content.formErrorMessage.errorRadioMessage' | translate } %}
+      {% set radiosConfig = radiosConfig | setAttribute('errorMessage', errorMessageObject) %}
+    {% endif %}
+
+    {{ govukRadios(radiosConfig) }}
+    {{ govukButton({
```

Comment:
I dont think this is the button we want here. We use this on pages where it is unrecoverable and direct the user to gov.uk

Reply:
Do you mean the actual button or just the text in it?

Reply:
Mainly the text. But also to keep consistency with the buttons we have on other pages where we use html over the nunjucks. Seems to be a bit of inconsistency overall but we majoritly use html buttons for these type of form pages

Reply:
Ok, I have updated the button to reference `general.buttons.next` which is "Continue". The use of `govukButton` was based on [this comment](https://github.com/govuk-one-login/ipv-core-front/pull/1339#discussion_r1528933932). I don't know, but would've thought `govukButton` was designed to replace html buttons. I also noticed an inconsistency, so it might be a good tech-debt ticket to standardise buttons whichever way we choose

Reply:
Yeah I'd suggested using the macro - be good to gradually refactor towards using this everywhere. But @MikeCollingwood I think you'll need to add `id: submitButton` (required for the event listener which disables the button on click)

Reply:
So that prevents double clicks but doesnt prevent multiple clicks if i remember correctly which is why we implemented our own javascript to prevent it